embedx/config.schema.json

```
@@ -312,6 +312,225 @@
         }
       }
     },
+    "selfServiceSAMLProvider": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "title":"ID of the IdentityProvider",
+          "type": "string",
+          "examples": [
+            "activedirectory1"
+          ]
+        },
+        "label": {
+          "title": "Optional string which will be used when generating labels for UI buttons.",
+          "type": "string",
+          "examples": [
+            "Microsoft Active Directory"
+          ]
+        },
+        "public_cert_path": {
+          "title": "Public Certificate Path",
+          "description": "The Public Certificate for your SAML Messages",
+          "type": "string",
+          "format": "uri",
+          "examples": [
+            "file://path/to/cert",
+            "https://foo.bar.com/path/to/cert"
+          ]
+        },
+        "private_key_path": {
+          "title": "Private Key Path",
+          "description": "The Private Key for your SAML Messages",
+          "type": "string",
+          "format": "uri",
+          "examples": [
+            "file://path/to/key",
+            "https://foo.bar.com/path/to/key"
+          ]
+        },
+        "mapper_url": {
+          "title": "Jsonnet Mapper URL",
+          "description": "The URL where the jsonnet source is located for mapping the provider's data to Ory Kratos data.",
```

misc english wording

```suggestion
          "description": "Location for the jsonnet mapping between provider's data and Ory Kratos data",
```